src/Perimeterx.php

```diff
@@ -92,6 +92,7 @@ private function __construct(array $pxConfig = [])
                 'sdk_name' => 'PHP SDK v2.8.0',
                 'debug_mode' => false,
                 'perimeterx_server_host' => 'https://sapi-' . strtolower($pxConfig['app_id']) . '.perimeterx.net',
+                'captcha_script_host' => 'http://sample-go.pxchk.net:8081',
```

Comment:
Whats that? should it be on prod?

Reply:
i understand why  we did this, but we cant merge that until we have a proper server